integrated_channels/integrated_channel/learner_data.py

```diff
@@ -134,6 +133,52 @@ def collect_learner_data(self):
                 is_passing=is_passing,
             )
 
+    def _should_skip_enrollments_data_collection(self, enterprise_enrollment):
+        """
+        Specifies whether to skip data collection of the enterprise enrollment due to any configuration reason.
+
+        Reasons for skipping data collection may include data sharing consent not being granted,
+        or audit track data reporting being disabled for audit enrollments, etc..
```

Comment:
Sure, I've adopted this format, however left out the last line because this isn't really a "public" function and that line reads like it's meant for that.